tests/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 
 MACRO_AddTest(${CMAKE_CURRENT_SOURCE_DIR}/DART_Modal)
 MACRO_AddTest(${CMAKE_CURRENT_SOURCE_DIR}/DART_Metafor)
+MACRO_AddTest(${CMAKE_CURRENT_SOURCE_DIR}/FPM_Modal)
```

Comment:
Note that adding it here means the battery must work both on Linux and on Windows.

Reply:
Yes, FPM should work on windows.